### ISSUE TITLE
collectd causes Sailfish OS 3.3 upgrade to fail on XA2

### ISSUE THREAD

**Issue by CODeRUS** [rinigus/collectd#9]
take a look at: https://together.jolla.com/question/226291/xa2-update-to-33016-failed-with-systemdatascope-installed-collectd-remove-failure/

proposed change in  spec file:

change
`su nemo -c "systemctl --user ***"`
calls to
`systemctl-user *** ||:`

**Comment by rinigus** [comment]
Is there such command as `systemctl-user` in SFOS 3.3? As I cannot test it right now, is this expected to work for new and older SFOS releases:

```
su nemo -c "systemctl --user stop %{name}.service" || systemctl-user stop %{name}.service || true
```

**Comment by CODeRUS** [comment]
sure this essential helper present in all sfos versions

**Comment by rinigus** [comment]
Should be fixed in 80b7e88449acc26fe99fed4358556d9c01765052 . New collectd released at OpenRepos with this fix. Closing here, please reopen if it doesn't work for you
